File: Python/BonAppetit.py
def bonAppetit(bill, k, b):
    # Write your code here
    sumOfAll = sum(bill)
    discarded = sumOfAll - bill[k]
    annaShouldPay = discarded/2
    if (b - annaShouldPay == 0):
        print("Bon Appetit")
    else:
        print(int(b - annaShouldPay))

File: C#/BonAppetit.cs
public static void bonAppetit(List<int> bill, int k, int b)
{
    int sumOfAll = bill.Sum();
    int discarded = sumOfAll - bill[k];
    int annaShouldPay = discarded/2;
    if (b - annaShouldPay == 0){
        Console.WriteLine("Bon Appetit");
    }
    else{
        Console.WriteLine(b - annaShouldPay);
    }
}